benchflow-test-manager/application/src/main/java/cloud/benchflow/testmanager/scheduler/TestTaskScheduler.java

```diff
@@ -156,6 +161,18 @@ private synchronized void handleWaitingState(String testID) {
       // set state as ready
       testModelDAO.setTestState(testID, BenchFlowTestState.READY);
 
+      // update max running time timeout
```

Comment:
I need to update the maxRunning time since that is read when the test goes back to `RUNNING`. In the `WAITING` state that is the only place I know how much time has elapsed since the `TimeoutTask` was scheduled.